plugins/jamf/jamf.go

```diff
@@ -0,0 +1,99 @@
+package jamf
+
+import (
+	"encoding/json"
+	"github.com/blinkops/blink-http/consts"
+	"github.com/blinkops/blink-http/implementation/requests"
+	"github.com/blinkops/blink-http/plugins/connections"
+	"github.com/blinkops/blink-http/plugins/types"
+	blink_conn "github.com/blinkops/blink-sdk/plugin/connections"
+	"github.com/pkg/errors"
+	"net/http"
+	"strings"
+)
+
+const (
+	requestUrlParam = "REQUEST_URL"
+)
+
+type JamfPlugin struct{}
+
+func (p JamfPlugin) TestConnection(connection *blink_conn.ConnectionInstance) (bool, []byte) {
+	requestUrl, ok := connection.Data[consts.RequestUrlKey]
+	if !ok {
+		return false, []byte(consts.RequestUrlMissing)
+	}
+
+	if !strings.Contains(requestUrl, p.GetDefaultRequestUrl()) {
+		return false, []byte("invalid request url")
+	}
+
+	res, err := connections.SendTestConnectionRequest(requestUrl+"/api/v1/auth", http.MethodGet, nil, connection, p.HandleAuth)
+	if err != nil {
+		return false, []byte("Test connection failed. " + err.Error())
+	}
+
+	if res.StatusCode != http.StatusOK {
+		return false, []byte(consts.TestConnectionFailed + string(res.StatusCode))
```

Comment:
```suggestion
		return false, []byte(consts.TestConnectionFailed + fmt.Sprintf("%d", res.StatusCode))
```
string(res.StatusCode) gives a warning, you can read what it means